.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
lets not commit this file `git rm --cached .editorconfig`

Reply:
removed .editorconfig

Reply:
sorry wrong file @cavukovic  `git rm --cached .vscode/launch.json`